src/Text.mo

```diff
@@ -1,23 +1,42 @@
 /// Text values
 ///
-/// This type describes a valid, human-readable text. It does not contain arbitrary
-/// binary data.
+/// This type represents human-readable text as sequences of characters of type [`Char`](Char.html) .
+/// If `t` is a value of type `Text`, then:
+/// * `t.chars()` returns an _iterator_ of type `Iter<Char>` enumerating its characters from first to last.
+/// * `t.size()` returns the _size_ (or length) of `t` (and `t.chars()`) as a `Nat`.
+///
+/// This module defines additional operations on `Text` values.
 
+import Char "Char";
 import Iter "Iter";
 import Hash "Hash";
 import Prim "mo:prim";
 
 module {
 
-  /// Returns the concatenation of `x` and `y`, `x # y`.
-  public func concat(x : Text, y : Text) : Text =
-    x # y;
+  /// Conversion.
+  /// Returns the text value of size 1 containing the single character `c`.
+  public let fromChar : (c : Char) -> Text = Prim.charToText;
+
+  /// Conversion.
+  /// Creates an [iterator](Iter.html#type.Iter) that traverses the characters of the text `t`.
+  public func toIter(t : Text) : Iter.Iter<Char> =
+    t.chars();
```

Comment:
Is `Text.toIter(t)` preferred over `t.chars()`?

Reply:
Probably not in a situation like you're describing, but it's much nicer in higher order situations. Say you had a `t : ?Text` and you need a `?(Iter<Char>)`.

```
Option.map(Text.toIter, t) // more concise than:
Option.map(func (t1 : Text) : Iter<Char> = t1.chars(), t)

Reply:
That makes sense.